regenerate-templates.sh

```diff
@@ -1,9 +1,98 @@
-#!/bin/sh
-ENMASSE_DIR=${1:-$PWD}
-pushd $ENMASSE_DIR
+#!/usr/bin/env bash
+
+do_patch () {
+    PATCH_DIR=$1
+    PATCH_TARGET=$2
+    if [ -d "${PATCH_DIR}" ]
+    then
+        for patch in $(find ${PATCH_DIR} -type f -name "*.patch"); do
+            echo Applying patch ${patch}
+            patch --remove-empty-files --merge -f -d "${PATCH_TARGET}" -p1 < $patch
```

Comment:
Should it perhaps fail if patch did not apply ?

Reply:
One alternative is to add ‘set -e’ in top of file to always fail on sub command errors